developer/graphql-intro.md

```diff
@@ -0,0 +1,150 @@
+# Getting started with GraphQL
+
+## Install
+
+- Make sure you are on Reaction 1.11 and above.
+- Check your current Reaction version by running: `reaction -v`
+
+> Note: The GraphQL API is an experimental, alpha prototype. Do **not** use for production.
+
+## Your first query
+
+To try a simple query, open the GraphiQL interface by going to `http://localhost:3000/graphql` in your browser.
+
+In the right-hand query box, enter the following:
+
+```js
+{
+  primaryShopId
+}
+```
+
+You should see a successful response:
+
+```js
+{
+  "data": {
+    "primaryShopId": "cmVhY3Rpb24vc2hvcDpKOEJocTN1VHRkZ3daeDNyeg=="
+  }
+}
+```
+
+Running this query through the GraphiQL interface is the equivalent of running this [GET query](http://localhost:3000/graphql-alpha?query=%7B%0A%20%20primaryShopId%0A%7D) to `http://localhost:3000/graphql-alpha` with this query string (`query=%7B%0A%20%20primaryShopId%0A%7D`) in the browser.
```

Comment:
I don't think we are intentionally supporting GET requests for GraphQL. It might be working based on default settings of the packages we use, but we should only be documenting POST requests as the way to interact with the GraphQL endpoint.

Reply:
I'm just going to remove this section for now! I just used this as my own double-checking.